agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -228,6 +228,7 @@ class backhaul_manager : public btl::transport_socket_thread {
     std::string bssid_bridge_mac;
 
     std::unique_ptr<beerocks::agent_ucc_listener> m_agent_ucc_listener;
+    std::unordered_map<std::string, beerocks_message::sVapsList> m_vaps_map;
```

Comment:
Why is the key a std::string instead of sMacAddr? Is there a reason beyond that it was done wrong for the UCC listener, so we want to make the same mistake again?

I'm sorry if I sound a bit pedantic, but it's annoying me more and more that people keep on using strings for storing MAC addresses, and I had hoped that @tomereli would be one to give a good example...